src/app/services/courses.service.mock.ts

```diff
@@ -0,0 +1,21 @@
+import { of } from 'rxjs';
+
+const mockCourses = [ {
+  id: 1,
+  title: 'Video Course 1. Name 1',
+  description: 'Learn about where you can find course descriptions, what information they include, how they work, and details about various components of a course description. Course descriptions report information about a university or college\'s classes. They\'re published both in course catalogs that outline degree requirements and in course schedules that contain descriptions for all courses offered during a particular semester.',
+  duration: 59,
+  creationDate: new Date(),
+}, {
+  id: 2,
+  title: 'Video Course 2. Name 2',
+  description: 'Learn about where you can find course descriptions, what information they include, how they work, and details about various components of a course description. Course descriptions report information about a university or college\'s classes. They\'re published both in course catalogs that outline degree requirements and in course schedules that contain descriptions for all courses offered during a particular semester.',
+  duration: 94,
+  creationDate: new Date(),
+} ]
+
+export const MockCoursesService = {
+  getCourses() {
```

Comment:
please, use stream conventions as follovs:
getCourses$: Observable<Course[]>;
getCourses$$: Subject<Course[]>|BehaviourSubject<Course[]>|ReplaySubject<Course[]>; (any type of Subject)

Reply:
Thanks, good to know it